### PR TITLE
Adding KeywordRecognizer UWP quickstart.

### PR DESCRIPTION
**Pull request by glecaros**
## Purpose
* This change adds a KeywordRecognizer quickstart for UWP (C#)

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

### CODE REVIEW

File: quickstart/csharp/uwp/keyword-recognizer/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+This sample demonstrates how to recognize keywords with C# under the Universal Windows Platform using the Speech SDK.
+See the [accompanying article](https://docs.microsoft.com/azure/cognitive-services/speech-service/quickstart-csharp-uwp) on the SDK documentation page which describes how to build this sample from scratch in Visual Studio 2017.
+
+## Prerequisites
+
+* A Windows PC with Windows 10 Fall Creators Update (10.0; Build 16299) or later and with a working microphone.
+* [Microsoft Visual Studio 2017](https://www.visualstudio.com/), Community Edition or higher.
+* The **Universal Windows Platform development** workload in Visual Studio.
+* Note: processor target ARM or ARM64 is not yet supported.
```
Comment (jhakulin) [inline quickstart/csharp/uwp/keyword-recognizer/README.md:11]: >processor target ARM or ARM64 is not yet supported [](start = 8, length = 50)

is this correct?

Comment (glecaros) [reply quickstart/csharp/uwp/keyword-recognizer/README.md:11]: No, that isn't , missed that part. Will remove.

**Comment by jhakulin** [comment]
> using System;

Do we have copyrights in samples?

---
Refers to: quickstart/csharp/uwp/keyword-recognizer/helloworld/MainPage.xaml.cs:1 in 8f074b1. [](commit_id = 8f074b1f496f406777072a8bc2ff5e6d247d125f, deletion_comment = False)

File: quickstart/csharp/uwp/keyword-recognizer/helloworld/MainPage.xaml.cs [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+            }
+        }
+
+        private async void RecognizeKeywordButton_Click(object sender, RoutedEventArgs e)
+        {
+            RecognizeKeywordButton.IsEnabled = false;
+            
+            if (recognizer == null)
+            {
+                recognizer= new KeywordRecognizer(AudioConfig.FromDefaultMicrophoneInput());
```
Comment (jhakulin) [inline quickstart/csharp/uwp/keyword-recognizer/helloworld/MainPage.xaml.cs:87]: add space

File: quickstart/csharp/uwp/keyword-recognizer/helloworld/Package.appxmanifest [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+<Package
+  xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
+  xmlns:mp="http://schemas.microsoft.com/appx/2014/phone/manifest"
+  xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
+  IgnorableNamespaces="uap mp">
+
+  <Identity
+    Name="63fcf322-b0b3-40e7-a235-0ba65ecb0f8e"
+    Publisher="CN=Gerardo"
```
Comment (jhakulin) [inline quickstart/csharp/uwp/keyword-recognizer/helloworld/Package.appxmanifest:11]: Your name is here

File: quickstart/csharp/uwp/keyword-recognizer/helloworld/Package.appxmanifest [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    Version="1.0.0.0" />
+
+  <mp:PhoneIdentity PhoneProductId="63fcf322-b0b3-40e7-a235-0ba65ecb0f8e" PhonePublisherId="00000000-0000-0000-0000-000000000000"/>
+
+  <Properties>
+    <DisplayName>helloworld</DisplayName>
+    <PublisherDisplayName>Gerardo</PublisherDisplayName>
```
Comment (jhakulin) [inline quickstart/csharp/uwp/keyword-recognizer/helloworld/Package.appxmanifest:18]: also here

File: quickstart/csharp/uwp/keyword-recognizer/README.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+* Set the active solution configuration and platform to the desired values under **Build** \> **Configuration Manager**:
+  * On a 64-bit Windows installation, choose `x64` as active solution platform.
+  * On a 32-bit Windows installation, choose `x86` as active solution platform.
+* Press Ctrl+Shift+B, or select **Build** \> **Build Solution**.
+
+## Run the sample
+
+To debug the app and then run it, press F5 or use **Debug** \> **Start Debugging**. To run the app without debugging, press Ctrl+F5 or use **Debug** \> **Start Without Debugging**.
+
+When you see the `Enable microphone` button, click it. Then select the `Listen for Keyword`. At this point you can say the keyword **"Computer"**. Once the keyword is detected, you can start talking and once you are done you can click the button `Save to File` to save the audio that was just recorded to a file.
```
Comment (jhakulin) [inline quickstart/csharp/uwp/keyword-recognizer/README.md:29]: > Then select the `Listen for Keyword`. [](start = 54, length = 38)

is there a selection for "Listen for Keyword" ?

Comment (glecaros) [reply quickstart/csharp/uwp/keyword-recognizer/README.md:29]: It is a button

File: quickstart/csharp/uwp/keyword-recognizer/helloworld/Package.appxmanifest (hunk 4 above)
Comment (glecaros) [reply quickstart/csharp/uwp/keyword-recognizer/helloworld/Package.appxmanifest]: fixed

File: quickstart/csharp/uwp/keyword-recognizer/helloworld/Package.appxmanifest (hunk 3 above)
Comment (glecaros) [reply quickstart/csharp/uwp/keyword-recognizer/helloworld/Package.appxmanifest]: fixed

File: quickstart/csharp/uwp/keyword-recognizer/helloworld/MainPage.xaml.cs (hunk 2 above)
Comment (glecaros) [reply quickstart/csharp/uwp/keyword-recognizer/helloworld/MainPage.xaml.cs]: fixed

**Comment by glecaros** [comment]
> > using System;
> 
> Do we have copyrights in samples?
> 
> Refers to: quickstart/csharp/uwp/keyword-recognizer/helloworld/MainPage.xaml.cs:1 in [8f074b1](https://github.com/Azure-Samples/cognitive-services-speech-sdk/commit/8f074b1f496f406777072a8bc2ff5e6d247d125f). [](commit_id = [8f074b1](https://github.com/Azure-Samples/cognitive-services-speech-sdk/commit/8f074b1f496f406777072a8bc2ff5e6d247d125f), deletion_comment = False)

**Pull request merged**